.github/workflows/pr-preview.yml

```diff
@@ -56,7 +56,7 @@ jobs:
                   cp -r build/* public/docs/
             - name: Compute stage
               run: |
-                  echo "STAGE=$(echo -n ${{ github.ref_name }} | shasum | head -c 8)" >> $GITHUB_ENV
+                  echo "STAGE=z$(echo -n ${{ github.ref_name }} | shasum | head -c 7)" >> $GITHUB_ENV
```

Comment:
I would use a more useful prefix such as `pr-` from `preview` instead of `z`.

This will be a bit more helpful for debugging purposes when checking stuff in the logs or in the databases.